version/1/2/0/code_of_conduct.md

```diff
@@ -0,0 +1,23 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or political affiliation.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexual language or imagery
+* Derogatory comments
+* Personal attacks
+* Trolling or insulting comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses, without permission
+* Other unethical or unprofessional conduct.
```

Comment:
@postmodern:
> Isn't this the problem we're trying to solve? Historically, maintainers have not been good at identifying and punishing harassment;

Which is why we're giving them examples. But examples are just that, trying to create a comprehensive list of forbidden behaviors would be an exercise in futility, and the document itself would be as unreadable as a typical EULA.

> we're also leaving a loophole for those that claim they are being persecuted due to "reverse sexism".

That isn't a loophole, they can and will claim that anyway. In fact, they'll probably claim that the mere existence of a CoC is itself "reverse sexism".

> Also, I'm pretty sure your hypothetical example violates Twitter's ToS, as it crosses the line between parody and defamation/libel.

Don't make me get really creative, that will just give the wrong people new ideas to use. :bomb: